guides/common/assembly_disaster-recovery-by-virtualizing-your-projectserver.adoc

```diff
@@ -0,0 +1,7 @@
+include::modules/con_disaster-recovery-by-virtualizing-your-project-server.adoc[]
```

Comment:
```suggestion
include::modules/con_disaster-recovery-by-virtualizing-your-projectserver.adoc[]
```

Suggestion (non-blocking): Align the file name to the assembly/all other files in this assembly.